src/tabContent.scss

```diff
@@ -5,3 +5,61 @@
     margin-bottom: 10px;
     font-family: "Segoe UI VSS (Regular)","Segoe UI","-apple-system",BlinkMacSystemFont,Roboto,"Helvetica Neue",Helvetica,Ubuntu,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
   }
+
+  html, body {
+    background-color: transparent !important;
+    margin: 0;
+    padding: 0;
+  }
+
+  .wide {
+    position: absolute;
+    top: 60px;
+    left: 0;
+    width: 100%;
+    height: 96%;
+    border: 1px solid #ddd;
+    background-color: #F8F8F8;
+    box-shadow: 0 2px 6px 2px rgba(0, 0, 0, .05);
+  }
+
+  #protractor-ext-message p, .wide p {
+    padding: 0;
+    margin: auto 0;
+    font-family: arial, serif;
+    text-align: center;
+    position: relative;
+    top: 50%;
+    color: #ccc;
+    font-size: 30px;
+  }
+
+  .error-badge {
+    background-image: url("../icon.png");
```

Comment:
I couldn't find where the original repo had an `images/` directory at all, so I opted to just point this to the only image in the project at the moment.